src/backend/crew_ai/optimization/keyword_vector_store.py

```
@@ -188,6 +188,30 @@ def search(self, library_name: str, query: str, top_k: int = 3) -> List[Dict]:
+                row = conn.execute(
+                    "SELECT name, args, doc FROM kw_keywords "
+                    "WHERE library = %s AND name = %s",
+                    (library_name, name)).fetchone()
+            if not row:
+                return None
+            return {
+                "name": row[0],
+                "args": row[1] if row[1] is not None else [],
+                "description": row[2] or "",
+            }
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)